metabase/metabase
# Add event sidebar to the query builder

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds a basic sidebar to view and toggle visibility of timeline events
- Can edit and archive events
- Cannot add events from the sidebar currently, will be implemented in a follow-up PR

How to test:
- Create a question and add a breakout (summarize with grouping) by a date column. E.g. Orders and group by Created date.
- Click on the calendar icon in the bottom right corner. You should see the timeline list.
- It should be possible to check and uncheck timelines. Currently there is no effect on the chart itself.
- It should be possible to edit and archive timeline events.

<img width="965" alt="Screenshot 2022-03-10 at 22 59 59" src="https://user-images.githubusercontent.com/8542534/157745019-41bafb00-8e2a-47ea-bd07-ebfc3240b6b5.png">
